pkgs/desktops/deepin/library/gio-qt/default.nix

```diff
@@ -20,15 +22,23 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-dlY1CTlXywgGZUonBBe3cDwx8h2xXrPY6Ft/D59nlug=";
   };
 
-  nativeBuildInputs = [ cmake pkg-config wrapQtAppsHook ];
+  patchPhase = ''
+    # qt.qpa.plugin: Could not find the Qt platform plugin "minimal"
+    # A workaround is to set QT_PLUGIN_PATH explicitly
+    export QT_PLUGIN_PATH=${qtbase.bin}/${qtbase.qtPluginPrefix}
+    substituteInPlace CMakeLists.txt --replace "qhelpgenerator" "${qttools.dev}/bin/qhelpgenerator"
```

Comment:
Isn't this taken from PATH?

Reply:
Should add qttools.dev instead of qttools in `nativeBuildInputs`,  updated already